datalad_catalog/catalog/templates/dataset-template.html

```diff
@@ -123,6 +124,64 @@ <h3>Cite dataset</h3>
             <span v-show="invalid_doi"><em>Invalid DOI</em></span>
           </b-modal>
         </div>
+        <b-modal id="modal-4" size="lg" 
+          header-bg-variant="light"
+          footer-bg-variant="light"
+          body-bg-variant="light"
+          body-text-variant="dark"
+          ok-only>
+            <template #modal-header="{ close }">
+                <h3>Share the dataset</h3>
+            </template>
+            <span v-if="selectedDataset.alias">
+              <h6>Alias URL</h5>
+              <div class="shadow-sm p-3 mb-3 bg-white rounded">
+                <b-row>
+                  <b-col cols="11">
+                    <code id="alias_url">
+                      <!-- TODO: this url does not take into account if catalog is served from a subdirectory relative to root -->
+                      {{window.location.origin + '/dataset/' + selectedDataset.alias}}
+                    </code>
+                  </b-col>
+                  <b-col cols="1">
+                    <i id="copyBtnAliasURL" class="far fa-clone" @click="copyDatasetURL('alias')"></i>
+                    <b-tooltip target="copyBtnAliasURL" :show.sync="showCopyDatasetAliasTooltip" @shown="hideURLTooltipLater('alias')" triggers left title="Copied!">
+                    </b-tooltip>
+                  </b-col>
+                </b-row>
+              </div>
+            </span>
+            <h6>ID URL</h5>
```

Comment:
I wonder if "Versioned URL" might be a better name?

Reply:
Wouldn't that make more sense for "full URL"?